src/ServicePulse.Host/vue/src/components/monitoring/EndpointListRow.vue

```diff
@@ -92,7 +92,7 @@ function formatGraphDecimal(input, deci) {
   <div class="table-col">
     <div class="box-header">
       <div class="no-side-padding">
-        <D3Graph :type="'queue-length'" :isdurationgraph="false" :plotdata="endpoint.metrics.queueLength" :minimumyaxis="smallGraphsMinimumYAxis.queueLength" :avglabelcolor="'#EA7E00'" :metricsuffix="'MSGS'" :key="endpoint"></D3Graph>
+        <SmallGraph :type="'queue-length'" :isdurationgraph="false" :plotdata="endpoint.metrics.queueLength" :minimumyaxis="smallGraphsMinimumYAxis.queueLength" :avglabelcolor="'#EA7E00'" :metricsuffix="'MSGS'" />
```

Comment:
it forced an update on the entire component, rather than letting the component decide what part of it needs updating through reactive state. I don't know why it was there previously, but it wasn't nice